adapters/rubicon/rubicon_test.go

```diff
@@ -255,6 +262,30 @@ func TestAppendTracker(t *testing.T) {
 	}
 }
 
+func TestSetImpNative(t *testing.T) {
+	testScenarios := []rubiSetImpNativeTestErrorScenario{
+		{
+			request:       "{}",
+			impNative:     map[string]interface{}{"somekey": "someValue"},
+			expectedError: fmt.Errorf("unable to find imp in json data"),
+		},
+		{
+			request:       "{\"imp\":[]}",
+			impNative:     map[string]interface{}{"somekey": "someValue"},
+			expectedError: fmt.Errorf("unable to find imp[0] in json data"),
+		},
+		{
+			request:       "{\"imp\":[{}]}",
+			impNative:     map[string]interface{}{"somekey": "someValue"},
+			expectedError: fmt.Errorf("unable to find imp[0].native in json data"),
+		},
```

Comment:
Thanks for adding the tests. These look good. Can you please also add a success test case where the request has an imp with native object defined?

Reply:
This case in json scenario